app/components/icons/brands/Cargo.tsx

```diff
@@ -0,0 +1,388 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
To reduce the overall document size, this cargo icon could be reduced/compressed